modules/Samples/SamplesTests/Extensions/CodeleExtensionTests.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  CodeleExtensionTests.swift
+//  SamplesTests
+//
+//  Copyright 2022 Thoughtworks, Inc. All rights reserved
+//
+
+import XCTest
+@testable import Samples
+
+struct CodableData: Codable {
+    let data: String
+}
+
+final class CodeleExtensionTests: XCTestCase {
+    func testResultISNotEmptyWhenDataIsCodable() {
```

Comment:
`testResultISNotEmptyWhenDataIsCodable` to `testResultIsNotEmptyWhenDataIsCodable `

Reply:
done